packages/main/src/i18n/messagebundle.properties

```diff
@@ -32,25 +32,25 @@ AVATAR_GROUP_ARIA_LABEL_GROUP=Conjoined avatars.
 AVATAR_GROUP_MOVE=Press ARROW keys to move.
 
 #XACT: ARIA announcement for the badge
-BADGE_DESCRIPTION_BADGE=Badge
+TAG_DESCRIPTION_BADGE=Badge
```

Comment:
Since we renmed the component, maybe it is reduntant to have "TAG_DESCRIPTION_BADGE" and confusing. We can leave only "Tag description", uneless there is some use case that we need the badge description?

Reply:
I was thinking about this, and probably yes, but I will leave this to the team, as there will be some follow-up alignments - from what I heard one of the design sets could be even removed, and if it's "Set3", then the text in question will be naturally removed.

<img width="235" alt="Screenshot 2024-05-02 at 10 14 55" src="https://github.com/SAP/ui5-webcomponents/assets/15702139/27128a9e-2093-4451-90a5-91a580491549">
<img width="557" alt="Screenshot 2024-05-02 at 10 15 12" src="https://github.com/SAP/ui5-webcomponents/assets/15702139/10c162d6-55ed-4813-8d70-8a365511cef0">

So I suggest merging it now, the pure refactoring, and later you can remove it.